docs/src/components/Footer/AvatarList.astro

```diff
@@ -1,6 +1,12 @@
 ---
 // fetch all commits for just this page's path
-const path = "docs/" + Astro.props.path;
+
+export interface Props {
+  path: string;
+}
+
+let {path} = Astro.props as Props;
+path = 'docs/' + path;
```

Comment:
Not a blocking issue, but because we're talking about best practices: Props are usually treated as "constant" in the React, Svelte, etc. communities. `let ... props` feels a bit confusing, even though its technically defining a new variable called path.

Thoughts on just moving `docs/` into the two strings below?

```
const url = `https://api.github.com/repos/snowpackjs/astro/commits?path=docs/${path}`;
```

Reply:
Yeah that's fine, I didn't write this code, just refactored it to add the interfaces.

Reply:
totally, appreciate the improvements even if they're a bit out of scope :)